addons/base_import/models/base_import.py

```diff
@@ -626,6 +637,21 @@ def _parse_import_data(self, data, import_fields, options):
                 # We should be able to manage both case
                 index = import_fields.index(name)
                 self._parse_float_from_data(data, index, name, options)
+            elif field['type'] == 'binary' and name in import_fields:
+                index = import_fields.index(name)
+                for line in data:
+                    url_regex = re.compile(
+                        r'^ftp|http|https|smb?://'  # http:// or https:// or ftp:// or smb://
+                        r'(?:/?|[/?]\S+)$', re.IGNORECASE)
+                    if url_regex.match(line[index]):
+                        try:
+                            line[index] = self.binary2base64(line[index])
+                        except requests.exceptions.ConnectionError:
+                                raise ValueError(_("Check your connection to import from URL"))
+                        except requests.exceptions.Timeout:
+                            raise ValueError(_("Server time out while importing from the URL, please try after some time"))
+                        except Exception:
+                            raise ValueError(_("Something went wrong with URL, please check the URL!"))
```

Comment:
There's basically no point to these errors. There is little value in treating these three cases differently, but there is value in:

1. noting the field was interpreted as a URL
2. printing the URL (field value) which caused an issue so that the user can easily find it in the file they are trying to import